README.md

```diff
@@ -3,6 +3,8 @@
 [![🧪 QGIS tests](https://github.com/qgis/QGIS/actions/workflows/run-tests.yml/badge.svg)](https://github.com/qgis/QGIS/actions/workflows/run-tests.yml?query=branch%3Amaster+event%3Apush)
 [![Docker Status](https://img.shields.io/docker/automated/qgis/qgis.svg)](https://hub.docker.com/r/qgis/qgis/tags)
 [![Build Status](https://dev.azure.com/qgis/QGIS/_apis/build/status/qgis.QGIS?branchName=master)](https://dev.azure.com/qgis/QGIS/_build/latest?definitionId=1&branchName=master)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/qgis/QGIS/badge)](https://securityscorecards.dev/viewer/?uri=github.com/qgis/QGIS)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/1581/badge)](https://www.bestpractices.dev/projects/1581)
```

Comment:
22% don't look spectacular. Do we want to put pressure on us to invest into following this approach?

Reply:
@NathanW2 Would be the person who could give me the ability to update y'all's report. I think your score could be _much_ higher since it was done last done in 2018.

Reply:
I think @NathanW2 is a bit away from contributing to the project these days, which is fine. What do we need exactly from Nathan ? Credentials or ownership transfer ? 
Or could we create another entry and do not bother Nathan?

Reply:
We have two options for the OpenSSF Best Practices Badge:
- He'd either have to delete [the entry](https://www.bestpractices.dev/en/projects/1581) attached to this repo
- Add an additional user ID to that existing entry. In this case all he'd have to do is add my user ID by opening the "Basics" menu and going to "(Advanced) What other users have additional rights to edit this badge entry?", type `+30764`, click Save and then I could edit that entry for y'all.

I'm content with just having the OpenSSF Scorecard Badge since that's technically less fudgeable and doesn't require someone manually updating the criterion. However it does provide a roadmap/explanations of the steps required to get a higher score and is less intrusive than applying the Scorecard CLI GitHub Action.

Reply:
Hello all.  Thanks for the ping sorry I had totally forgotten that I had even done this back in the day. 

@gregorywaynepower I'm happy to take what ever action you would like here. 

I think I watched a presentation about this the other day and thought "we should do that for QGIS"... not even remembering I had started one :) Thanks past Nathan

Reply:
@NathanW2 I am more than willing to edit the existing report you've created if you give me the ability to do so. I appreciate the work you've done so far to get the ball rolling and I don't want your hard work to go to waste.

Reply:
@gregorywaynepower done.  Added +3076 hope that works for you.

Reply:
@NathanW2 One number shy. `+30764` is my user ID. If you want to remove that user ID it's `-3076`

Reply:
@gregorywaynepower oops what a noob error.  Fixed now.  Should be good to go.